src/components/todolist/todolist.tsx

```diff
@@ -0,0 +1,292 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
I hope there are no errors when removing these

Reply:
Yes, this just gives some warnings. Like for unused variables. Because of ES-Lint extension

Reply:
Yes no errors, these are warnings like for unused variables, because of ESLint extension